src/components/icons/BaseIcon.tsx

```diff
@@ -1,15 +1,17 @@
 export interface BaseIconProps {
     className?: string;
+    isFilled?: boolean;
 }
 
 const BaseIcon: React.FunctionComponent<BaseIconProps> = ({
     className,
     children,
+    isFilled = false,
 }) => {
     return (
         <svg
             viewBox="0 0 24 24"
-            fill="none"
+            {...(isFilled ? {} : { fill: 'none' })}
```

Comment:
Nit: I don't think the empty object is needed. I _think_ you can instead do `...( isFilled && { fill: 'none' } )` and it would work the same?

Reply:
Ah yeah looks like that works, didn't know that, thank you!